tests/ui/suspicious_command_arg_space.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    std::process::Command::new("echo").arg("hello world").spawn().unwrap();
+    std::process::Command::new("echo").arg("-n hello").spawn().unwrap();
+    std::process::Command::new("cat").arg("--number file").spawn().unwrap();
```

Comment:
nit: also some tests about string arguments that _can_ have spaces (i.e., doesn't start with a `-`)

Reply:
Like the first line here, with `"hello world"`? Or do you mean something else?

Reply:
Oh sorry I meant something that tests the alphanumeric constant. Some garbage like `-4@sjiTu== foo`